euphonic/cli/show_sampling.py

```diff
@@ -41,7 +46,8 @@ def main(params: Optional[list[str]] = None) -> None:
 
     if args.sampling == 'golden-square':
```

Comment:
This is definitely a space for a `match` at some point! (Not this PR, since it's not being changed.)